drivers/led_strip/Kconfig.ws2812

```diff
@@ -17,6 +17,14 @@ config WS2812_STRIP_SPI
 	  The SPI driver is portable, but requires significantly more
 	  memory (1 byte of overhead per bit of pixel data).
 
+config WS2812_STRIP_SPI_PREAMBLE_SUPPORT
+	bool "WS2812 LED strip SPI preamble support"
+	default n
```

Comment:
Ideally, the default value should be set based on whether a device tree SPI attribute is present (like `spi-idle-high` for example). This SPI behavior should be described somehow in device tree.

Reply:
It would need review from SPI maintainers but don't really see why it should need more than that

Reply:
@tbursztyka @teburd would you be ok with introduction of such a flag?

Reply:
Ok, as a compromise I have added this line
`default y if SOC_FAMILY_NXP_IMXRT`
Which will automatically enable this config if the driver is used on a IMXRT family SOC, its not a perfect solution, but it is functional and non intrusive.